examples/vpa-rules.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: autoscaling.k8s.io/v1
+kind: VerticalPodAutoscaler
+metadata:
+  name: rule-evaluator
+  namespace: gmp-system
+spec:
+  resourcePolicy:
+    containerPolicies:
+    - containerName: evaluator
+      controlledResources:
+        - memory
+      maxAllowed:
+        memory: 4Gi
```

Comment:
Any reason to keep the `maxAllowed`? I ended up removing this from the collector VPA because the limit just felt kind of arbitrary.